<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactsEditAddress</class>
 <widget class="QDialog" name="ContactsEditAddress">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit contact address...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/contacts</normaloff>:/contacts</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="street_label">
       <property name="text">
        <string notr="true">Street:</string>
       </property>
       <property name="buddy">
        <cstring>street</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="street"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="po_box_label">
       <property name="text">
        <string notr="true">Post-office box:</string>
       </property>
       <property name="buddy">
        <cstring>po_box</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="po_box"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="extension"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="zip_code_label">
       <property name="text">
        <string notr="true">Postal/ZIP code:</string>
       </property>
       <property name="buddy">
        <cstring>zip_code</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="zip_code"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="extension_label">
       <property name="text">
        <string notr="true">Extension:</string>
       </property>
       <property name="buddy">
        <cstring>extension</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="city_label">
       <property name="text">
        <string notr="true">City:</string>
       </property>
       <property name="buddy">
        <cstring>city</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="state_label">
       <property name="text">
        <string notr="true">State/Province:</string>
       </property>
       <property name="buddy">
        <cstring>state</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="country_label">
       <property name="text">
        <string notr="true">Country/Region:</string>
       </property>
       <property name="buddy">
        <cstring>country</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="city"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="state"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="country"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ContactsEditAddress</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ContactsEditAddress</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
